apps/organizations/tests/test_org_models.py

```diff
@@ -66,7 +67,7 @@ def test_org_deletion_cascades_when_no_contributions_downstream(self):
         self.assertFalse(RoleAssignment.objects.filter(id=ra_id).exists())
 
 
-class RevenueProgramTest(TestCase):
+class RevenueProgramTest:
```

Comment:
Believe this test class won't be picked up by pytest's test runner.
```suggestion
class TestRevenueProgram:
```